server/run-tests-with-grcov.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# install deps
+cargo install grcov
+rustup component add llvm-tools-preview
```

Comment:
This doesn't need to be run every time...

Reply:
These commands don't redownload or reinstall if the dependencies already exist on the system. Including them here just reduces manual installation. Can remove if wanted.

Reply:
I think it's better if you added a section to the server README about code coverage with instructions to install this + how to run.

I'd probably also remove `open` from the script and just print out `open THE_PATH.html`